egs/heroico/s5/README

```diff
@@ -0,0 +1,4 @@
+ heroico
+Scripts  for Kaldi recipe for Spanish using the Heroico corpus.
```

Comment:
Some information about the Heroico corpus here would be nice.  E.g. how much train and test data is there, and what kind of speech does it contain?